src/pages/workspaces/workspace/WorkspaceContainer.ts

```diff
@@ -405,17 +422,20 @@ export const wrapWorkspace =
           namespace,
           name,
           activeTab,
-          showTabBar,
           workspace,
           refreshWorkspace,
           title: _.isFunction(title) ? title(props) : title,
           breadcrumbs: breadcrumbs(props),
-          topBarContent: topBarContent && topBarContent({ workspace, ...props }),
           analysesData: { apps, refreshApps, runtimes, refreshRuntimes, appDataDisks, persistentDisks },
           storageDetails,
           refresh: async () => {
             await refreshWorkspace();
-            if (child.current?.refresh) {
+            if (
+              !!child?.current &&
```

Comment:
That's a lot of checks. I wonder if you can just call `_.isFunction`.

Reply:
I didn't realize that would work with type checking.
It's only a marginal improvement, but it is a little better, particularly when combining with `_.isObject`:
`_.isObject(child?.current) && 'refresh' in child.current && _.isFunction(child.current.refresh`